src/marqo/tensor_search/tensor_search.py

```diff
@@ -1287,12 +1293,15 @@ def search(config: Config, index_name: str, text: Optional[Union[str, dict]],
         if approximate is None:
             approximate = True
 
+        marqo_index = index_meta_cache.get_index(config=config, index_name=index_name)
+        text_query_prefix = determine_text_prefix(text_query_prefix, marqo_index.index_settings, "text_query_prefix")
```

Comment:
there's no point in determining the prefix in both `search` and `vector_text_search`. We can just do this in `vector_text_search`

Reply:
Oh yeah, thanks for catching that. I forgot to delete the one in `search`